### PR TITLE
test(appeals): added e2e tests for viewing case history events assign…

### PR DESCRIPTION
**Pull request by RHARV018**
…ing case officer and validation

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [x] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/e2e/cypress.config.js [hunk 1]
```diff
@@ -46,6 +46,7 @@ module.exports = defineConfig({
 		defaultCommandTimeout: 30000,
 		pageLoadTimeout: 60000,
 		experimentalModifyObstructiveThirdPartyCode: true,
+		experimentalRunAllSpecs: true,
```
Comment (PeterBridgerKC) [inline appeals/e2e/cypress.config.js:49]: Why was this change included?

File: appeals/e2e/cypress/page_objects/caseDetailsPage.js [hunk 2]
```diff
@@ -134,6 +134,16 @@ export class CaseDetailsPage extends Page {
 		this.elements.manageAgreementToChangeDescriptionEvidence().click();
 	}
 
+	verifyTableCellTextCaseHistory(answer) {
```
Comment (PeterBridgerKC) [inline appeals/e2e/cypress/page_objects/caseDetailsPage.js:137]: This new method should go into the "Verfifications" area of the file, so next to `verifyAnswerSummaryValue()` a little further down

File: appeals/e2e/cypress.config.js (hunk 1 above)
Comment (RHARV018) [reply appeals/e2e/cypress.config.js:49]: this is to make it so that we can run all tests in 1 go if needed

File: appeals/e2e/cypress/page_objects/caseDetailsPage.js (hunk 2 above)
Comment (RHARV018) [reply appeals/e2e/cypress/page_objects/caseDetailsPage.js]: good point - thank you, have moved in second commit

**Review by fabfab-kc — DISMISSED** [review]
Looks good!

**Review by fabfab-kc — APPROVED** [review]
LGTM!
